.projenrc.js

```diff
@@ -80,14 +83,21 @@ const project = new web.ReactTypeScriptProject({
 // synthesize project files before build
 // see https://github.com/projen/projen/issues/754
 const buildTask = project.tasks.tryFind("build");
+buildTask.spawn(project.packageTask);
+buildTask.prependSpawn(project.testTask);
 buildTask.prependSpawn(project.compileTask);
 buildTask.prependExec("npx projen");
-buildTask.spawn(project.packageTask);
 
 // npm tarball will only include the contents of the "build"
 // directory, which is the output of our static website.
-project.npmignore?.addPatterns("!/build");
-project.npmignore?.addPatterns("/public");
+project.npmignore.addPatterns("!/build");
+project.npmignore.addPatterns("/public");
+
+// assemblies used for tests
+project.npmignore.addPatterns("src/__assemblies__");
```

Comment:
These are test fixtures, correct? Maybe just call them `__fixtures__`? 

It's just not clear that these are intended for tests since tests are co located here. 

Alternatively consider putting tests under `src/__tests__` but I am actually quite fond of the co-location

Reply:
I'm also quite fond of the co-location. changed to `__fixtures__/assemblies` since we might have other fixtures as well.